package/doc/sphinx/source/documentation_pages/analysis/wbridge_analysis.rst

```diff
@@ -0,0 +1 @@
+.. automodule:: MDAnalysis.analysis.hbonds.wbridge_analysis
```

Comment:
Please build the docs locally, check manually, and confirm with a comment that your docs pass your visual inspection.

Reply:
@orbeckst I have checked and can confirm that all links works correctly.